app/components/wallet/layouts/WalletWithNavigation.scss

```diff
@@ -13,3 +13,13 @@
   height: calc(100% - 40px);
   overflow: auto;
 }
+
+:global(.YoroiModern) {
+  .navigation {
+    height: 50px;
```

Comment:
this should probably be a variable that you import from the other file instead of a magic number

Reply:
yeah, I agree. This number is used in multiple parts and there are some calculations using it, so it shouldn’t be hardcoded.

Reply:
@SebastienGllmt @nicarq 
https://github.com/Emurgo/yoroi-frontend/pull/636/commits/ddfdbe509a0c4ce1afe98c710176c24766af6990

Initially i though i should make a const, but don't remember why I quit that idea :)